src/plugins/dashboard/public/events.ts

```diff
@@ -7,3 +7,4 @@
  */
 
 export const DASHBOARD_LOADED_EVENT = 'dashboard_loaded';
+export const SAVED_OBJECT_LOADED_TIME = 'saved_object_loaded_time';
```

Comment:
nit: can we use a name that is less likely to clash? Something like `dashboard_saved_objects_list_loaded`?

Reply:
Oh! Nevermind! I noticed that you are planning to add `meta.saved_object_type` to be able to differentiate.